docker-compose.yml

```diff
@@ -10,6 +10,7 @@ services:
       - POSTGRES_USER=${DB_USERNAME:-postgres}
       - POSTGRES_PASSWORD=${DB_PASSWORD}
       - POSTGRES_DB=ACAB_DB
+      - ROSTER_SOURCE=${ROSTER_SOURCE:-https://techblocsea.sfo3.digitaloceanspaces.com/spd-lookup/latest}
```

Comment:
Same for the reference here. And we probably don't need a default here as well if it's in the .env.